bench/iced-x86/src/main.rs

```diff
@@ -1,58 +1,78 @@
+use iced_x86::{Decoder, DecoderOptions, Instruction};
+#[cfg(feature = "formatter")]
+use iced_x86::{SpecializedFormatter, SpecializedFormatterTraitOptions};
 use std::fs::File;
 use std::io::Read;
 use std::io::Seek;
 use std::io::SeekFrom;
 
-use iced_x86::{Decoder, DecoderError, DecoderOptions, Instruction};
-
-#[cfg(not(DISAS_BENCH_NO_FORMAT))]
-use iced_x86::{Formatter, NasmFormatter};
+#[cfg(feature = "formatter")]
+struct TraitOptions;
+#[cfg(feature = "formatter")]
+impl SpecializedFormatterTraitOptions for TraitOptions {
+    const ENABLE_DB_DW_DD_DQ: bool = false;
+    unsafe fn verify_output_has_enough_bytes_left() -> bool {
+        false
+    }
+}
```

Comment:
Hm, not sure how I feel about this. I think this benchmark should reflect standard configurations that most people are expected to run. I'd imagine the amount of people who would actively choose to opt into disabling buffer overrun checks is rather low.

Reply:
The buffer is much bigger (300 bytes) than any disassembled text so that's not possible. IIRC, diStorm doesn't check its output buffer so it's similar.

This only saves ~5-6MB/s on my system so it's not a big increase.